Core/include/Acts/Seeding/CompositeSpacePointLineSeeder.hpp

```diff
@@ -9,24 +9,206 @@
+    /// @brief Toggle whether an initial t0 fit shall be executed
+    bool fastSegFitWithT0{false};
+    /// @brief Maximum number of iterations in the fast segment fit
+    std::size_t nMaxIter{100};
+    /// @brief Precision cut off in the fast segment fit
+    double precCutOff{1.e-6};
+  };
+
+  struct SeedParameters {
+    /// @brief: Theta of the line
     double theta{0.};
     /// @brief Estimated intercept
     double y0{0.};
     /// @brief Uncertainty on the angle
     double dTheta{0.};
     /// @brief Uncertainty on the intercept
     double dY0{0.};
+    /// @brief Line parameters
+    Line_t line{};
+    /// @brief Vector of radial signs of the valid hits
+    std::vector<int> solutionSigns{};
+    /// @brief Iterations to obtain the seed with fast fit
+    std::size_t nIter{0};
+    /// @brief Seed chi2
+    double chi2{0.};
+    /// @brief Number of straw hits on the seed
+    std::size_t nStrawHits{0};
+    /// @brief Estimated parameters from pattern
+    Line_t::ParamVector patternParams{};
+
+    /// @brief Stringstream output operator
+    friend std::ostream& operator<<(std::ostream& ostr,
+                                    const SeedParameters& sol) {
+      return sol.print(ostr);
+    }
+    /// @brief Prints the seed solution to the screen
+    std::ostream& print(std::ostream& ostr) const;
+  };
+
+  /// @brief Seed object returned by the seeder. The seed contains the initial parameter estimate w.r.t
+  ///         to the central plane surface inside the chamber.///Note* the
+  /// parameter q/p is set to zero and does not serve any purpose here.
+  ///
+  ///         Further the seed contains the list of associated measurements. If
+  /// the seeder is run in the fast 2D fit mode, the number of iterations w.r.t.
+  /// this fit procedure is appended as well.
+
+  /// @brief Cache of all solutions seen thus far
+  template <CompositeSpacePointContainer Cont_t>
+  struct SeedSolution : public SeedParameters {
+    SeedSolution(const SeedParameters& pars) : SeedParameters(pars) {};
+    SeedSolution() {};
+    /// @brief Used hits in the seed
+    Cont_t seedHits{};
+  };
+
+  template <CompositeSpacePointContainer Cont_t,
+            CompositeSpacePointSorter<Cont_t> Splitter_t,
+            CompositeSpacePointContainer CalibCont_t,
+            CompositeSpacePointCalibrator<Cont_t, CalibCont_t> Calibrator_t>
+  struct SeedOptions {
+    // @brief Splitter holding the straw and strip hits
+    std::unique_ptr<Splitter_t> splitter{};
+    using Sp_t = SpacePoint_t<Cont_t>;
+    // @brief Good hit selector
+    Selector_t<Sp_t> selector{};
+    AbortSelector_t abortSelector{};
+    // @brief Calibrator
+    const Calibrator_t* calibrator{nullptr};
+    // @brief Experiment specific calibration context
+    const CalibrationContext* calibContext{};
+
+    std::size_t upperLayer{0};
+    std::size_t lowerLayer{0};
+    std::size_t lowerHitIndex{0};
+    std::size_t upperHitIndex{0};
+
+    uint m_signComboIndex{0};
```

Comment:
```suggestion
    std::size_t signComboIndex{0};
```
the other data members don't have any m_ prefixes either